static/js/zamboni/devhub.js

```diff
@@ -6,7 +6,7 @@ $(document).ready(function () {
   $('#edit-addon').exists(initEditAddon);
 
   // Poll for suppressed email removal updates.
-  $('#verification_pending').exists(function () {
+  $('.verify-email#verification_pending').exists(function () {
     setTimeout(function () {
       window.location.reload();
     }, 10_000);
```

Comment:
Let's bump to 30 seconds to go with the adjusted timeout suggested above

Reply:
Done.